src/mlpack/core/data/confusion_matrix.hpp

```diff
@@ -0,0 +1,48 @@
+/**
+ * @file confusion_matrix.hpp
+ * @author Jeffin Sam
+ *
+ * Function for confusion matrix
```

Comment:
Do you think you could add some more details here?

Reply:
added some details, do suggest what should i add more